eng/pipelines/runtime.yml

```diff
@@ -62,1443 +62,21 @@ extends:
         - template: /eng/pipelines/common/evaluate-default-paths.yml
 
       #
-      # Build CoreCLR checked
-      # Only when CoreCLR is changed
+      # PAL Tests
       #
       - template: /eng/pipelines/common/platform-matrix.yml
         parameters:
-          jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-          buildConfig: checked
-          platforms:
-          - linux_x86
-          - linux_x64
-          - linux_arm
-          - linux_arm64
-          - linux_riscv64
-          - linux_musl_arm
-          - linux_musl_arm64
-          - linux_musl_x64
-          - osx_arm64
-          - tizen_armel
-          - windows_x86
-          - windows_x64
-          - windows_arm64
-          jobParameters:
-            testGroup: innerloop
-            condition: >-
-              or(
-                eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
-                eq(dependencies.evaluate_paths.outputs['SetPathVars_runtimetests.containsChange'], true),
-                eq(variables['isRollingBuild'], true))
-
-      #
-      # Build the whole product using GNU compiler toolchain
-      # When CoreCLR, Mono, Libraries, Installer and src/tests are changed
-      #
-      - template: /eng/pipelines/common/platform-matrix.yml
-        parameters:
-          jobTemplate: /eng/pipelines/common/global-build-job.yml
-          buildConfig: checked
-          platforms:
-          - gcc_linux_x64
-          jobParameters:
-            testGroup: innerloop
-            nameSuffix: Native_GCC
```

Comment:
Why are so many legs  including the gcc one being removed?

Reply:
This is temporary to make testing easier. I'll revert this before I make the PR non-draft

Reply:
I've reverted this portion of the change and this PR is now ready for review.